### PR TITLE
Fix for https://github.com/thomseddon/koa-oauth-server/issues/29

### PR DESCRIPTION
**Pull request by wehriam**

### CODE REVIEW

File: lib/grant.js [hunk 1]
```diff
@@ -466,10 +466,8 @@ function sendResponse (done) {
 
   if (this.refreshToken) response.refresh_token = this.refreshToken;
 
-  this.res
-    .set('Cache-Control', 'no-store')
-    .set('Pragma', 'no-cache')
-    .jsonp(response);
+  this.res.set({'Cache-Control':'no-store', 'Pragma': 'no-cache'});
```
Comment (thomseddon) [inline lib/grant.js]: Just missing a single space :) Could it please be:
```
this.res.set({'Cache-Control': 'no-store', 'Pragma': 'no-cache'});
```

**Comment by thomseddon** [comment]
Sorry for delay - could you add the extra space?

**Comment by wehriam** [comment]
@thomseddon - Done.

**Comment by thomseddon** [comment]
Thanks :+1:

**Comment by wehriam** [comment]
Could do an incremental release to incorporate this fix?

**Comment by wehriam** [comment]
@thomseddon could you do an incremental release to push this fix?

**Comment by thomseddon** [comment]
Sorry will do

**Comment by thomseddon** [comment]
Done in 2.4.1

**Pull request merged**